tests/__init__.py

```diff
@@ -116,8 +119,22 @@ async def _create_db():
     return db
 
 
+class _ActiveDirectoryLDAPServer(LDAPServer):
+    def getRootDSE(self, request, reply):
```

Comment:
Would be nice to add a comment to this saying what customization this is making.